lib/overlays/menu_overlay.dart

```diff
@@ -73,7 +75,11 @@ class MenuOverlay extends StatelessWidget {
                 }),
             // PrivacyPolicyButton
             Expanded(
-              flex: 2,
+              flex: 1,
+              child: ProgressBar(),
```

Comment:
should the progressbar be on the Menuscreen?
@adrianpfeiffer, @tguelenman

Reply:
Yes